packages/scene-composer/src/components/panels/scene-settings/MatterportIntegration.tsx

```diff
@@ -22,7 +22,8 @@ export const MatterportIntegration: React.FC = () => {
   const setSceneProperty = useStore(sceneComposerId)((state) => state.setSceneProperty);
 
   const [dirty, setDirty] = useState(false);
-  const [matterportModelId, setMatterportModelId] = useState(getSceneProperty(KnownSceneProperty.MatterportModelId));
+  const matterportModelId = useRef(getSceneProperty(KnownSceneProperty.MatterportModelId));
```

Comment:
whys it a ref? we set it as a ref, and then force re-renders anyway with dirty flags?

Reply:
useState was not working for me at the time when I was working on it. I wanted to verify the checks on changes, so created a PR with these changes. Have reverted it to useState and now it is working as expected.